docs/product-oracle-weblogic-installation-and-configuration-guide/src/main/asciidoc/controller-wls-install-proc.adoc

```diff
@@ -4,7 +4,8 @@
 To use the {KIE_SERVER} REST API or Java Client API to interact with {KIE_SERVER}, you can install the {KIE_SERVER} Controller with {WEBLOGIC}. The {KIE_SERVER} Controller manages {KIE_SERVER} configuration in a centralized way so that you can use the Controller to create and maintain containers and perform other server-level tasks.
 
 .Prerequisites
-* An {WEBLOGIC} instance is configured as described in xref:wls-configure-proc[].
+* The {WEBLOGIC} instance is configured as described in xref:wls-configure-proc[].
+* {KIE_SERVER} is installed on the {WEBLOGIC} instance.
```

Comment:
@sutaakar, my understanding was that the Controller is for the Process/Decision Server, which is therefore a prerequisite. In a different application server, the process server is still a prerequisite, because the Controller interacts with the Process Server as usual, but the Process Server is just integrated with the WLS/WAS. I haven't gathered that the Controller is to interact directly with the WAS/WLS. Am I wrong?